posts/urls.py

```diff
@@ -1,9 +1,18 @@
 from django.urls import path
+from .views import (
+    PostListView,
+    PostDetailView,
+    ResumeCreateView,
+    ResumeUpdateView,
+)
 from . import views
 
 
 urlpatterns = [
-    path('', views.feed, name='posts-feed'),
+    path('', PostListView.as_view(), name='posts-feed'),
+    path('post/<int:pk>/', PostDetailView.as_view(), name='post-detail'),
+    path('post/new/resume', ResumeCreateView.as_view(), name='resume-create'),
+    path('post/<int:pk>/update/', ResumeUpdateView.as_view(), name='resume-update'),
```

Comment:
Because the update page of the resume and other types of posts is a different html file I think that the URL should be: `post/resume/<int:pk>/update`

Reply:
fixed